smart-contracts/test/Unlock/upgrades.js

```diff
@@ -173,10 +173,17 @@ contract('Unlock / upgrades', async (accounts) => {
             it('PublicLock version is set', async () => {
               if (versionNumber >= 1) {
                 // Version numbers were introduced to PublicLock with v1
-                const version = await lock.publicLockVersion()
-                if (versionNumber == 2) {
+                const templateVersion = await lock.publicLockVersion()
+                const version =
+                  typeof templateVersion !== 'number'
+                    ? templateVersion.toNumber()
+                    : templateVersion
+                if (versionNumber === 2) {
                   // version 2 had a bug: we forgot to bump the lock version
                   assert.equal(version, 1)
+                } else if (versionNumber === 9) {
```

Comment:
At some point we should make these better because in practice Unlock should be able to deploy any version of PublicLock (this is going to be part of the work on making PublicLock upgradable!)